api/helpers/get-rematch-games.js

```diff
@@ -1,6 +1,13 @@
 // Pull the logic into func outside fn to allow recursion w/o bumping into weirdness w/ sails' exits
 async function getRematchGames (game, priorRematchGames = []) {
-  const rematchGames = [game, ...priorRematchGames];
+  const rematchGames = [game, ...priorRematchGames].map((matchGame) => {
+    return {
+      ...matchGame,
+      // Get p0 and p1 id, whether populated or not
+      p0: matchGame.p0?.id ?? matchGame.p0,
```

Comment:
Should these be added as keys if they don’t exist? I don’t how or when that case would occur though.

Reply:
I think we always want to p0 and p1 properties to exist, but to be nullable. We shouldn't hit this part and have them missing because they are set when users join the game and this is going through finished games, but just in case I think it's better that the keys exist so games can be assumed to have a p0 and p1 attribute even if null (meaning there is no player)